app/assets/javascripts/table_row_click.coffee

```diff
@@ -1,3 +1,4 @@
+# FIXME: This may be broken now
```

Comment:
I don't think this is broken... by dropping `window.` in the function definition, the click handler can't "see" the function to call, because coffeescript wraps the code in an anonymous function, making all function definitions not applied to the global scope, like would normally happen with vanilla javascript.